paymentsheet/src/androidTest/java/com/stripe/android/paymentelement/EmbeddedPaymentElementTestRunner.kt

```diff
@@ -0,0 +1,120 @@
+@file:OptIn(ExperimentalEmbeddedPaymentElementApi::class)
+
+package com.stripe.android.paymentelement
+
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.ui.Modifier
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.PaymentConfiguration
+import com.stripe.android.link.account.LinkStore
+import com.stripe.android.networktesting.NetworkRule
+import com.stripe.android.paymentsheet.CreateIntentCallback
+import com.stripe.android.paymentsheet.PaymentSheet
+import kotlinx.coroutines.test.runTest
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+internal class EmbeddedPaymentElementTestRunnerContext(
+    private val embeddedPaymentElement: EmbeddedPaymentElement,
+    private val countDownLatch: CountDownLatch,
+) {
+    suspend fun configure() {
+        embeddedPaymentElement.configure(
+            intentConfiguration = PaymentSheet.IntentConfiguration(
+                mode = PaymentSheet.IntentConfiguration.Mode.Payment(amount = 5000, currency = "USD")
+            ),
+            configuration = EmbeddedPaymentElement.Configuration.Builder("Example, Inc.").build()
+        )
+    }
+
+    fun confirm() {
```

Comment:
An alternative would be to put a "buy" button on screen, but it would just call this, so it seems fair.